easybuild/easyconfigs/g/GlobalArrays/GlobalArrays-5.8-gomkl-2021a.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'GlobalArrays'
+version = '5.8'
+
+homepage = 'https://hpc.pnl.gov/globalarrays'
+description = "Global Arrays (GA) is a Partitioned Global Address Space (PGAS) programming model"
+
+toolchain = {'name': 'gomkl', 'version': '2021a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/GlobalArrays/ga/releases/download/']
+sources = ['v%(version)s/ga-%(version)s.tar.gz']
+checksums = ['64df7d1ea4053d24d84ca361e67a6f51c7b17ed7d626cb18a9fbc759f4a078ac']
+
+builddependencies = [('Perl', '5.32.1', '-minimal')]
+
+configopts = ' --with-mpi --enable-i8 '
+configopts += '--with-blas8="-L$MLROOT/lib/intel64 -Wl,--no-as-needed -lmkl_gf_ilp64 -lmkl_sequential '
```

Comment:
```suggestion
configopts += '--with-blas8="-L$MKLROOT/lib/intel64 -Wl,--no-as-needed -lmkl_gf_ilp64 -lmkl_sequential '
```

Reply:
Thanks for finding that typo! I have corrected it.